curriculum/challenges/english/25-front-end-development/workshop-recipe-ingredient-converter/67353880ed1f52ba28c8eb56.md

```diff
@@ -13,7 +13,7 @@ Time to see your curried function in action. Here's how you might call our examp
 const dinner = tikkaMasala("curry powder")("tofu")("onion");
 ```
 
-Declare a `gramsResult` variable, and assign it the result of using your curried function to convert `2` `cup`s to `gram`.
+Declare a `gramsResult` variable, and assign it the result of using your curried function to convert 2 cups to `gram`.
```

Comment:
`2` and `cup` are both values that are in the code, just removing the backticks is not going to work

Reply:
```suggestion
Declare a `gramsResult` variable, and assign it the result of using your curried function to convert `2` units of the measure `cup` to `gram`.
```